app/models/module_item.rb

```diff
@@ -160,8 +158,9 @@ def page_name
   # @return [Boolean]
   delegate :valid?, to: :model
 
+  # @return [Boolean]
   def notes?
-    model.notes? if model.respond_to?(:notes?)
+    model.respond_to?(:notes?) ? model.notes? : false
```

Comment:
the other way works and is easier to read, why the change? Not a problem, just interested in why the preference....

Reply:
The change was to ensure it just a boolean, a later commit pruned back more after inheriting. I will revisit this bit now.

Reply:
I have used delegate, which means we could actually have the learning log on common pages too if we wanted